ember_debug/utils/ember/runloop.js

```diff
@@ -10,7 +10,7 @@ const keys = ['cancel', 'debounce', 'join', 'later', 'scheduleOnce'];
 
 if (EmberRunloop) {
   module = EmberRunloop;
-  _backburner = EmberRunloop._backburner;
+  _backburner = EmberRunloop._backburner || EmberRunloop.backburner;
```

Comment:
Until at least 3.24, we're working with `backburner`, but in more recent versions it's `_backburner`.